optimizn/combinatorial/anneal.py

```diff
@@ -72,39 +60,6 @@ def anneal(self, n_iter=100000, reset_p=1/10000):
                 self.update_best(self.new_candidate, self.new_cost)
                 print("Best cost updated to:" + str(self.new_cost))
 
-
```

Comment:
Did these move elsewhere? What is the reasoning?

Reply:
Yeah, I moved those to a separate OptProblem class, which both the SimAnnealProblem class and the BnbProblem class will inherit from. Certain methods (cost, cost_delta, get_candidate) are applicable to both, so thought it would be better placed in the OptProblem class